openfga/v1/authzmodel.proto

```diff
@@ -59,6 +61,26 @@ message TypeDefinition {
       example: "{\"relations\":{\"reader\":{\"union\":{\"child\":[{\"this\":{}},{\"computedUserset\":{\"object\":\"\",\"relation\":\"writer\"}}]}},\"writer\":{\"this\":{}}}}"
     }
   ];
+
+  // A map whose keys are the name of the relation and whose value is the TypeInformation for that
+  // relation.
+  map<string, TypeInformation> type_info = 3;
+}
+
+message TypeInformation {
+  repeated RelationReference assignable_relations = 1;
+}
+
+// RelationReference represents a relation of a particular object type (e.g. 'document#viewer').
```

Comment:
it could also be something like `person`, no?

Reply:
yeah